dlp/api/Snippets.Tests/UpdateStoredInfoTypesTests.cs

```diff
@@ -64,8 +64,13 @@ public void TestUpdate()
             finally
             {
                 // Delete the created buckets.
-                storage.DeleteBucket(bucketOutput.Name, new DeleteBucketOptions { DeleteObjects = true });
-                storage.DeleteBucket(bucket.Name, new DeleteBucketOptions { DeleteObjects = true });
+                storage.DeleteBucket(bucketOutput.Name);
+                var objects = storage.ListObjects(bucket.Name);
+                foreach (var ob in objects)
+                {
+                    await storage.DeleteObjectAsync(bucket.Name, ob.Name);
+                }
+                await storage.DeleteBucketAsync(bucket.Name);
```

Comment:
Please go back to using `DeleteObjects = true` as that should definitely work here. And if it doesn't let me know as soon as it happens (do not rerun tests so I can see the failure).

Reply:
Alternatively consider using RandomBucketFixture and BucketCollector here the same as it's done in #2305

Reply:
RandomBucketFixture seems to be working fine. I did not get any errors while running it on local. So, I have changed the sample with RandomBucketFixture. Thanks.